deploy/roles/eod/tasks/main.yaml

```diff
@@ -245,3 +252,163 @@
   when: custom_repo
   tags:
     - eod
+
+- name: Create Xconfig directory 
+  file: 
+    path: "{{ eod_Xconfig_directory }}"
+    state: directory
+    mode: 0755
+  tags: 
+    - eod 
+
+- name: Set /etc/gdm/gdm.schema file
+  template:
+    src: gdm.schemas.j2
+    dest: /etc/gdm/gdm.schemas
```

Comment:
@olivermel It looks like this template doesn't contain any variables.  Are some of these settings supposed to be changeable?  How does this file differ from the default file?